Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+      "fields": [
+        { "name": "Datum", "type": "BossEggMobileSpawner::Datum" },
+        { "name": "Altitude", "type": "float32" },
+        { "name": "Unknown1", "type": "uint8" },
+        { "name": "Unknown2", "type": "uint8" },
+        { "name": "AppearOffset1D", "type": "float32" },
+        { "name": "Speed", "type": "float32" },
+        { "name": "Unknown3", "type": "uint8" }
+      ]
+    },
+    "BossFerrisWheelBaseSpawner": {
+      "fields": [
+        { "name": "SearchRadius", "type": "float32" }
+      ]
+    },
+    "BossFerrisWheelSpawner": {
+      "parent": "BossFerrisWheelBaseSpawner",
+      "fields": [
+        { "name": "ShotSpeed", "type": "float32" },
+        { "name": "ShotLifeTime", "type": "float32" },
+        { "name": "ShotHomingWait", "type": "float32" },
+        { "name": "ShotChaseTime", "type": "float32" },
+        { "name": "ShotRotateSpeed", "type": "float32" },
+        { "name": "ShotCollisionRadius", "type": "float32" }
+      ]
+    },
+    "BossCoffeeCupSpawner": {
+      "parent": "BossFerrisWheelBaseSpawner",
+      "fields": [
+        { "name": "AttackTime[0]", "type": "float32" },
+        { "name": "AttackTime[1]", "type": "float32" },
+        { "name": "AttackTime[2]", "type": "float32" }
+      ]
+    },
+    "ObjCoffeeCupSpawner": {
+      "parent": "ObjGondolaBaseSpawner",
+      "fields": [
+        { "name": "PixieType[0]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[1]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[2]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[3]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[4]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[5]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[6]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[7]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[8]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[9]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[10]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[11]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[12]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[13]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[14]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[15]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[16]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[17]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[18]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[19]", "type": "ObjCoffeeCupSpawner::PixieType" }
+      ]
+    },
```

Comment:
Parameters named based on in-game functionality and parameters from other objects with similar functionality, specifically `enmMotora` for the naming of the 20 `PixieType` parameters. The type of the 20 parameters have been given the same type as to not clutter the enums data, considering they do the same.